packages/headless/src/controllers/core/generated-answer/headless-core-generated-answer.ts

```diff
@@ -241,8 +139,29 @@ export interface GeneratedAnswerAnalyticsClient {
   logGeneratedAnswerCollapse: () => CustomAction;
 }
 
+export interface GeneratedAnswerProps {
+  initialState?: {
```

Comment:
I think we should define the type of `initialState` in a separate interface (e.g., `GeneratedAnswerPropsInitialState`) rather than inline here.

Reply:
You resolved this comment but I don't see the change 😅

Reply:
https://github.com/coveo/ui-kit/pull/4129/files#diff-7409be0b72c2931d8b6f67a543483583ca497601bafd32145eaebab92ec8bc45R142

Did I resolve before my push ?

That would be just like me.

Reply:
Or else you meant doing it another way.